Havit.Blazor.Components.Web.Bootstrap/Buttons/HxButton.razor

```diff
@@ -5,6 +5,7 @@
            Placement="@TooltipPlacement"
            Container="@TooltipContainer"
            CssClass="@TooltipCssClass"
+           Trigger="@GetTooltipTrigger()"
```

Comment:
Also add `Settings="TooltipSettingsEffective"` to pass the new settings to the tooltip!